addons/l10n_at/data/account_tax_template.xml

```diff
@@ -1701,17 +2055,22 @@
                   'repartition_type': 'tax',
                   'minus_report_line_ids':  [ref('tax_report_line_l10n_at_tva_line_5_2')],
                 }),
+                (0,0,{
+                  'factor_percent': -100,
+                  'repartition_type': 'tax',
+                  'account_id': ref('chart_at_template_3520'),
+                }),
             ]"/>
         </record>
         <!-- Vorsteuern betreffend die geschuldete, auf dem Abgabenkonto
             verbuchte Einfuhrumsatzsteuer (§ 12 Abs. 1 Z 2 lit. b) -->
         <record id="account_tax_template_purchase_xx_code083" model="account.tax.template">
             <field name="name">VST_083 verbuchte EUst. (§ 12 Abs. 1 Z 2 lit. b)</field>
-            <field name="description">VSt. 20%</field>
+            <field name="description">20%</field>
             <field name="chart_template_id" ref="l10n_at_chart_template" />
             <field name="sequence">400</field>
             <field name="type_tax_use">none</field>
-            <field eval="20" name="amount" />
+            <field eval="100" name="amount" />
```

Comment:
@wtaferner What is the point in +100 +100 -100 -100 if you can just do 0 +100?

Reply:
Maybe because 100 != 0...  So this is reporting only?

Reply:
It is about having those move lines properly next to each other...it is basically two account moves with debit/credit pairs combined into one move (basically how accounting is normally done in its simple form...one at a time without split or combination)...for a reason we do not want to cancel out the amount on the same account and have a zero account move line left. Those two lines reconcile which each other instead of not exist at all as it is an awkward zero line.

It is more about auditing than reporting, but is also important for this specific case in reporting.

This is by the way some feature request I do have as it is requested from many accountants and tax lawyers...

Namely, having the direct opposite accounts for each debit/credit line (`many2many`) to know how it evolved to the final debit/credit which is missing in Odoo to look at an isolated line and know what was going on, even though you can retrieve somehow the all the other lines based on the move name, but sometimes the other move lines of a move can be completely unrelated, depending how you grouped your move lines into a move.

Reply:
@wtaferner It would be good to understand at least why this tax is that special.  Possibly this solution does the trick, but it is still weird.

Reply:
@jco-odoo 
How? I tried to explain it several times and tried it to configure or make it possible to the accountants in several ways and this was the best way how they properly encode and understand this tax in Odoo and also keep track of what the use case is asking for.

If you have several hours to deep dive in person, I could try to show you all the stuff I have and my customers have to handle, but honestly, I do not believe it is worth the effort. Whenever there is a better way to do it, believe me, I will do it and suggest, especially as I am constantly thriving to improve stuff and get feedback from accountants and tax lawyers in Austria to push the localization and Odoo forward. This is the Austrian localization, I am pretty sure there are other weird countries and use cases out there 😜

Reply:
**\*\*redacted rant\*\***

@jco-odoo @tsb-odoo 
I am here to help, you might want to start here, but do not expect me to understand that I am wrong about something which has been elaborated over several years...

**KZ 061**
Vorsteuern betreffend die entrichtete Einfuhrumsatzsteuer (§ 12 Abs. 1 Z 2 lit. a)
**KZ 083**
Vorsteuern betreffend die geschuldete, auf dem Abgabenkonto verbuchte Einfuhrumsatzsteuer (§ 12 Abs. 1 Z 2 lit. b)

https://www.ris.bka.gv.at/NormDokument.wxe?Abfrage=Bundesnormen&Gesetzesnummer=10004873&Paragraf=12

https://findok.bmf.gv.at/findok?execution=e4s1#RZ_1874a
https://formulare.bmf.gv.at/service/formulare/inter-Steuern/pdfd/2023/U30a.pdf